cost/azure/unused_volumes/azure_unused_volumes.pt

```diff
@@ -774,88 +775,84 @@ end
 ###############################################################################
 
 define delete_volumes($data, $param_take_snapshot, $$rs_optima_host) return $all_responses do
-  # Change "No" to "Yes" to enable CWF debug logging
-  $log_to_cm_audit_entries == "No"
-
-  $$debug = $log_to_cm_audit_entries == "Yes"
   $status_code=''
-  $all_responses = []
+  $$all_responses = []
   foreach $item in $data do
     sub on_error: skip do
-      call sys_log("In for item =",to_s($item))
       if $param_take_snapshot == "true"
-        call create_snapshot($item) retrieve $status_code
-        if to_s($status_code)=="202"
+        call create_snapshot($item) retrieve $response
+        # We don't expect to get anything other than Succeeded, but just in case before deleting disk lets check
+        if $response["body"]["properties"]["provisioningState"] == "Succeeded"
           call delete_unattached_disk($item['id'])
         else
-          call sys_log("Azure create snapshot error statusCode: ",to_s($status_code))
```

Comment:
Was this intentionally removed? If so, should the above else statement still be there?

Reply:
Fixed in 667b048.  good catch!